kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md

```diff
@@ -10,7 +10,7 @@ The following prerequisites must be handled prior to running the create domain s
 * Make sure the WebLogic operator is running.
 * The operator requires WebLogic Server 12.2.1.3.0 with patch 29135930 applied. The existing WebLogic Docker image, `store/oracle/weblogic:12.2.1.3`, was updated on January 17, 2019, and has all the necessary patches applied; a `docker pull` is required if you pulled the image prior to that date. Refer to [WebLogic Docker images](../../../../../site/weblogic-docker-images.md) for details on how to obtain or create the image.
 * Create a Kubernetes namespace for the domain unless the intention is to use the default namespace.
-* In the same Kubernetes namespace, create the Kubernetes persistent volume where the domain home will be hosted, and the Kubernetes persistent volume claim for the domain. For samples to create a PV and PVC, see [Create sample PV and PVC](../../create-weblogic-domain-pv-pvc/README.md).
+* In the same Kubernetes namespace, create the Kubernetes persistent volume (PV) where the domain home will be hosted, and the Kubernetes persistent volume claim (PVC) for the domain. For samples to create a PV and PVC, see [Create sample PV and PVC](../../create-weblogic-domain-pv-pvc/README.md). By default, the `create-domain.sh` scripts create a domain with the `domainUID` set to `domain1` and expect the PVC `domain1-weblogic-sample-pvc` is present. You can create `domain1-weblogic-sample-pvc` using [create-pv-pvc.sh](../../create-weblogic-domain-pv-pvc/create-pv-pvc.sh) with an inputs file that has the `domainUID` set to `domain1`.
```

Comment:
By default, the `create-domain.sh` scripts create (is there more than one `create-domain.sh` script?) If not, then the sentence should be, By default, the `create-domain.sh` script creates...

and expect the PVC `domain1-weblogic-sample-pvc` is present -> and expects the PVC `domain1-weblogic-sample-pvc` to be present.